partner_deduplicate_by_ref/wizards/partner_merge.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
```

Comment:
```suggestion
from odoo import fields, models
```
runbot is red, maybe that is because of this